src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -55,17 +55,16 @@
 import org.kohsuke.stapler.DataBoundConstructor;
 
 import java.io.IOException;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
can you please avoid star imports?

You may want to configure your IDE to not do this